salt/modules/saltutil.py

```diff
@@ -1608,11 +1622,16 @@ def runner(name, arg=None, kwarg=None, full_return=False, saltenv='base', jid=No
             prefix='run'
         )
 
-    return rclient.cmd(name,
-                       arg=arg,
-                       kwarg=kwarg,
-                       print_event=False,
-                       full_return=full_return)
+    if asynchronous:
+        master_key = salt.utils.master.get_master_key('root', __opts__)
+        low = {'arg': arg, 'kwarg': kwarg, 'fun': name, 'key': master_key}
+        return rclient.cmd_async(low)
```

Comment:
`rclient.cmd` and `rclient.cmd_async` seem to have inconsistent interfaces - a quick check and `LocalClient.cmd_async` accepts full parameters, but other `cmd_async`s accept a dictionary...

Reply:
What's the deal with `low`? Is it normal elsewhere or something?